README.md

```diff
@@ -2,52 +2,61 @@
 
 ## Summary
 
-This is a proposal to enable users and authors to group multiple resources into a single link. An agent navigating to such a link will load and present all its resources in a grouping UI. We're calling such a link a "page collection":
+It's common for users browsing the web to open and refer to multiple pages in a single journey. For example, when doing research on a
+complex topic, comparing products when shopping, or planning a trip or outing. Browsers have adapted to this by introducing tabbed UIs, and
+later allowing grouping of related tabs. However, linking on the web remains 1-to-1; one link opens one page. This makes it difficult
```

Comment:
Restructuring suggestion (totally up to you to use or not) : You say browsers adapted for multi-page/-site journeys by introducing tabs and groups. Then maybe we can say "It's also common that during these complex, multi-page journeys, users want to include others by sharing the content and getting their thoughts to make the decision together. Yet- linking on the web, the only mechanism of sharing, remains 1-to-1; one link opens one page."

Reply:
Thanks, added this reworded slightly for brevity